src/main/resources/templates/pdf/legalStatementGrantOfProbate.html

```diff
@@ -281,12 +272,22 @@
             {% endif %}
             {% endfor %}
             {% else %}
-            <p>None.</p>
+                {% if case_details.case_data.titleAndClearingType != "TCTPartAllRenouncing" and case_details.case_data.titleAndClearingType != "TCTPartSuccAllRenouncing" %}
+                    <p>None.</p>
+                {% endif %}
             {% endif %}
         </td>
     </tr>
     <tr>
         <td  colspan="2">
+            {% if case_details.case_data.titleAndClearingType == "TCTPartAllRenouncing" %}
+            <p><span>The profit-sharing {{ case_details.case_data.pluralProfitSharingTextForLegalStatement }} in the firm {{ case_details.case_data.nameOfFirmNamedInWill }}
+                at the date of death of the deceased have renounced probate.</span></p>
+            {% elseif case_details.case_data.titleAndClearingType == "TCTPartSuccAllRenouncing" %}
+            <p><span>The profit-sharing {{ case_details.case_data.pluralProfitSharingTextForLegalStatement }} in the firm {{ case_details.case_data.nameOfSucceededFirm }}
+                that had succeeded to and carried on the practice of the firm {{ case_details.case_data.nameOfFirmNamedInWill }}
+                    at the date of death of the deceased have renounced probate.</span></p>
+            {% endif %}
```

Comment:
super nitpicky (there may be a better way to balance the content across lines - the 'important' part is having all the text consistently indented in a block):

```suggestion
            {% if case_details.case_data.titleAndClearingType == "TCTPartAllRenouncing" %}
                <p><span>The profit-sharing {{ case_details.case_data.pluralProfitSharingTextForLegalStatement }} in the firm {{ case_details.case_data.nameOfFirmNamedInWill }}
                at the date of death of the deceased have renounced probate.</span></p>
            {% elseif case_details.case_data.titleAndClearingType == "TCTPartSuccAllRenouncing" %}
                <p><span>The profit-sharing {{ case_details.case_data.pluralProfitSharingTextForLegalStatement }} in the firm {{ case_details.case_data.nameOfSucceededFirm }}
                that had succeeded to and carried on the practice of the firm {{ case_details.case_data.nameOfFirmNamedInWill }}
                at the date of death of the deceased have renounced probate.</span></p>
            {% endif %}
```